libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts

```diff
@@ -58,36 +50,25 @@ import { WattToastService } from '@energinet-datahub/watt/toast';
 @Component({
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush,
-  providers: [
-    DbAdminAssignableUserRolesStore,
-    DbAdminInviteUserStore,
-    {
-      provide: STEPPER_GLOBAL_OPTIONS,
-      useValue: { showError: true },
-    },
-  ],
+  providers: [DbAdminAssignableUserRolesStore, DbAdminInviteUserStore],
   selector: 'dh-invite-user-modal',
   templateUrl: './dh-invite-user-modal.component.html',
   styleUrls: ['./dh-invite-user-modal.component.scss'],
```

Comment:
suggestion: looks like some of the styles here were moved to `watt-stepper`. Consider removing the duplicated styling.